VisualPinball.Unity/VisualPinball.Unity.Editor/Utils/HandlesUtils.cs

```diff
@@ -0,0 +1,75 @@
+﻿using UnityEditor;
+using UnityEngine;
+using VisualPinball.Unity.VPT;
+
+namespace VisualPinball.Unity.Editor.Utils
+{
+	class HandlesUtils
+	{
+		public delegate void OnPositionChange(Vector3 newPosition, params object[] plist);
+
+		public static void HandlePosition(Vector3 position, ItemDataTransformType type, Quaternion rotation, OnPositionChange onChange, params object[] plist)
+		{
+			HandlePosition(position, type, rotation, 0.2f, 0.0f, onChange, plist);
+		}
+
+		public static void HandlePosition(Vector3 position, ItemDataTransformType type, Quaternion rotation, float handleSize, float snap, OnPositionChange onChange, params object[] plist)

```

Comment:
I think we should treat this method as if it was one of Unity's built in handles, i.e. follow their conventions so surrounding code looks similar, and other folks coming to it later will have less to grok.
So basically I guess what I'm saying is, instead of having a change callback, just return the value out of the func, and leave the change check stuff to the caller.

Reply:
Now that all the drag point code is part of an ItemInspector, can we just get rid of DataFieldUtils and simplify back to the way it was before?

Reply:
Yup i removed the DataFieldUtils and removed the callback system from the HandlesUtils. Maybe if some EditorWindow not directly heriting from the ItemInspector need some Datafield edition, we could find a way to avoid code duplication at this moment.